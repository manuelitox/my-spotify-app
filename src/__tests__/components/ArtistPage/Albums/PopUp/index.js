import React from 'react'
import { shallow } from 'enzyme'

import PopUpAlbums from 'components/ArtistPage/Albums/PopUp/'

describe('Component | PopUpAlbums', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <PopUpAlbums isOpen togglePopUp={ jest.fn() } />
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})