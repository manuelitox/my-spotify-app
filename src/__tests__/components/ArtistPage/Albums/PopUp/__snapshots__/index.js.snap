// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | PopUpAlbums should render a component 1`] = `
<OnClickOutside(PopUpContainer)
  eventTypes={
    Array [
      "mousedown",
      "touchstart",
    ]
  }
  excludeScrollbar={false}
  isOpen={true}
  outsideClickIgnoreClass="ignore-react-onclickoutside"
  preventDefault={false}
  stopPropagation={false}
  togglePopUp={[MockFunction]}
>
  <Title
    color="#262D45"
    insidePopUp={true}
  >
    Albums
  </Title>
  <styled.div
    inPopup={true}
  >
    <Album
      cover="path/to/image.jpg"
      id="123abc"
      key="0"
      numberTracks={11}
      releaseDate="11/11/2019"
      title="album name"
    />
  </styled.div>
</OnClickOutside(PopUpContainer)>
`;
